content/docs/dvclive/index.md

```diff
@@ -1,18 +1,25 @@
 # dvclive
 
-[`dvclive`](/doc/dvclive) is an open-source Python library for monitoring the
-progress of metrics during training of machine learning models.
+[`dvclive`](https://github.com/iterative/dvclive) is an open-source Python
+library for monitoring the progress of metrics during training of machine
+learning models. You can simply
+[install it](https://github.com/iterative/dvclive#installation) with `pip`.
 
-Dvclive integrates seamlessly with [DVC](https://dvc.org/) and the logs it
-produces can be fed as `dvc plots`. However, `dvc` is not needed to work with
-`dvclive` logs, and since they're saved as easily parsable TSV files, you can
-use your preferred visualization method.
+DVCLive integrates seamlessly with `DVC` and the logs/summaries it produces can
+be fed as `dvc plots`/`dvc metrics`.
 
-We have created Dvclive with two principles in mind:
+However, `DVC` is not needed to work with `dvclive` logs/summaries, and since
+they're saved as easily parsable TSV/JSON files, you can use your preferred
+visualization method.
 
-- **No dependencies.** While you can install optional integrations for various
-  frameworks, the basic `dvclive` installation doesn't have requirements besides
-  [Python](https://www.python.org/).
-- **DVC integration.** `dvc` recognizes when its being used along with
-  `dvclive`. This enables useful features automatically, like producing model
-  training summaries, among others.
+We have created DVCLive with Git and MLOps principles in mind:
+
+- **Codification of data**. Tracked metrics are stored in readable text files
+  that can be versioned by Git or other version control tools.
+- **Distributed**. No services or servers are required. Metrics are stored in a
+  Git repository as text files, or pointers to files in `DVC` storage.
+- **GitOps API**. logs/summaries are generated through `DVC` using Git commit
+  SHAs or branch names, e.g.: `dvc plots diff --target logs master`.
```

Comment:
Can we add the image here from https://github.com/iterative/dvclive?

Reply:
You could also further simplify and clean up this example:
* `--target dvclive` instead of `--target logs` (I think the default path changed at some point).
* Not sure if we need to specify a revision. It adds more complexity but also introduces potentially helpful syntax. If we do, let's make sure it matches between the text and the image. Right now it shows `master` in text but `HEAD` in the plot.